lib/charms/tls_certificates_interface/v2/tls_certificates.py

```diff
@@ -742,6 +759,29 @@ def generate_certificate(
             extension.value,
             critical=extension.critical,
         )
+
+    # Filter out default extensions that were overridden by the caller. Without this step we'd get
```

Comment:
why isn't this filtering approach used for the other "default" extensions we are setting, like `AuthorityKeyIdentifier`, `SubjectKeyIdentifier` and `SubjectAlternativeName`?

Reply:
Removed `additional_extensions` arg.